src/Translations.elm

```diff
@@ -93,7 +93,7 @@ translationSetFor : TranslationKey -> TranslationSet
 translationSetFor translationKey =
     case translationKey of
         PageTitle ->
-            { english = "Is the metro borked?"
+            { english = "Is the metro broken?"
```

Comment:
This was supposed to be a joke: https://twitter.com/isfotis/status/1099618118102585344

Maybe it makes no sense to most people...

Reply:
But yeah, maybe it's better to fix it.